install.sh

```diff
@@ -1,18 +1,25 @@
 #!/bin/bash
-set -e
-EVALAI_TOKEN=${1?"Usage: $0 <EVALAI_TOKEN>"}
-
 
 echo "=== CREATING CONDA ENV 'clcomp21'"
-#source `conda info --base`/bin/activate base
-conda env create -f environment.yaml || true
+# source `conda info --base`/bin/activate base
+conda env create -f environment.yaml
+# echo "`conda shell.bash hook`"
 conda activate clcomp21
 
 echo "==== Installing Python packages (this may take a while) "
 pip install -r requirements.txt
 # NOTE: Required step, as evalai causes a dependency conflict with `Sequoia`.
 pip install --upgrade wandb
 
-evalai set_token "$EVALAI_TOKEN"
+
+EVALAI_TOKEN=`cat evalai_token.txt`
+if [ $EVALAI_TOKEN == "YOUR_TOKEN_HERE" ]; then
```

Comment:
no, you can't rerun the install script later (bc `conda env create` can only run once). The token MUST be present. Check that it's present at the start of this file (i.e. undo the removal of lines 2-4)

Reply:
Conda create can run more than once, it will simply fail to create the env, which doesn't interrupt the script from running

Reply:
No it can't. You removed the `|| true` in your commit.